common_operations/force_merge.json

```diff
@@ -0,0 +1,25 @@
+{
+    "operation": {
+        "operation-type": "force-merge",
+        "request-timeout": {{ request_timeout | default(60) | tojson }}{%- if max_num_segments is defined %},
```

Comment:
It might be better to make this explicit, since it applies only to the force-merge operation.  So calling it something like `force_merge_request_timeout` will make it clear it does not apply to all operations in general.

I'd do this in the next set of changes, just so this PR can go in as is.